llm/gguf.go

```diff
@@ -506,7 +508,7 @@ func writeGGUFArray[S ~[]E, E any](w io.Writer, t uint32, s S) error {
 	return binary.Write(w, binary.LittleEndian, s)
 }
 
-func WriteGGUF(ws io.WriteSeeker, kv KV, ts []Tensor) error {
+func WriteGGUF(ws io.WriteSeeker, kv KV, ts []Tensor, fn func(api.ProgressResponse)) error {
```

Comment:
Do you know if it's possible to add a test to `WriteGGUF`?

Reply:
Not sure if there exists one but there definitely should be. I assumed it would have been tested in whichever PR brought in `WriteGGUF`.  If there isn't one, I think it should be added but would be out of scope for this PR imo.